gradle/tools.gradle

```diff
@@ -19,3 +19,13 @@ task getDockerImageTag {
         println("${project.properties['dockerImageTag']}")
     }
 }
+
+task generateDocs (type: JavaExec) {
```

Comment:
Could we also add verification in the PR that the docs are published and bundled as expected? 
steps - https://github.com/DataBiosphere/terra-cli/blob/mc-feature/CONTRIBUTING.md

Reply:
I added some notes to the description. But just to confirm what you mean. Are you suggesting to add a Java unit test to verify the content of the distribution folder? as I don't see any similar test in the repo verifying the gradle tasks themselves and whether they are setup correctly.

Reply:
this works